lib/honeybadger/plugins/resque.rb

```diff
@@ -64,6 +64,7 @@ def payload_class_with_honeybadger
         end
 
         execution do
+          return unless Honeybadger.config[:'exceptions.enabled']
```

Comment:
If the plugin is exceptions-only, you could do this as a requirement block (a little cleaner):

```ruby
requirement { config[:'exceptions.enabled'] }
```

Reply:
I decided to go with the early return just in case we opted to add Insights-related code later. I didn't want to come along and Insights later without noticing the requirement block and then wonder why Insights wasn't working. :)